Jupyter/EzPlot_Example/EzPlot_Example.html

```diff
@@ -3,8 +3,6 @@
 title: EzPlot_Example Tutorial
 ---
 
-# EzPlot_Example Tutorial
-[download iPython Notebook](EzPlot_Example.ipynb)
```

Comment:
We need the download link

Reply:
I've added in html code to properly display the download link.  The code I removed previously should still be removed since it is markdown code that was not getting correctly converted to html and does not provide an actionable link to download the notebook (it just looks like a mistake on the html page).